terraform/templates/cluster-test/ct

```diff
@@ -44,11 +47,15 @@ done
 if [ -z "$DOCKER_IMAGE" ]
 then
       echo "Using master cluster test image"
-      $(aws ecr get-login --no-include-email --region us-west-2)
       DOCKER_IMAGE="853397791086.dkr.ecr.us-west-2.amazonaws.com/libra_cluster_test:master"
-      docker pull $DOCKER_IMAGE
+else
+      echo "Using $DOCKER_IMAGE image"
 fi
 
+echo "Logging in to ECR repo"
```

Comment:
Why did you make this change?

Reply:
We were only doing a `docker pull` for `:master` tag. 

With `cti`, when we start using the `:dev_$USER_pr_1234` tag, we will have to pull that as well because for different builds of the same pr, `cti` will use the same tag and hence I updated this code to always pull.

Reply:
I see.
One of usages is `--latest` flag that uses locally compiled latest image, so docker pull will fail on it
I suggest to just add docker pull to `-i` flag